app/brave_strings.grd

```diff
@@ -137,11 +137,14 @@ If you update this file, be sure also to update google_chrome_strings.grd. -->
         Brave
       </message>
       <if expr="is_win">
-        <message name="IDS_SXS_SHORTCUT_NAME" desc="Unused in Brave builds" translateable="false">
```

Comment:
These should be in the rebaes_l10n script since this file is generated.

Reply:
I'll handle it, but for future reference we never want to modify `.grd` files in patches
https://github.com/brave/brave-browser/blob/master/lib/l10nUtil.js#L87